README.md

```diff
@@ -22,8 +22,8 @@ Found a bug or have a new idea? Feel free to open [issues](https://github.com/va
 
 ## Authors
 
-- **Vatsal Joshi** - _Initial work_ - [vatz88](https://github.com/vatz88)
-- **Sangeeth Sudheer** - [sangeeth96](https://github.com/sangeeth96)
+-   **Vatsal Joshi** - _Initial work_ - [vatz88](https://github.com/vatz88)

```

Comment:
Why there's a diff in readme if nothing is changed?

Reply:
I just did rebase and dealt with report_chennai. Strange

Reply:
Can you try again. Ideally there should be only one commit and one file change which is the excel report.